addons/website_slides/models/slide_channel.py

```diff
@@ -265,15 +265,18 @@ def _get_default_enroll_msg(self):
         domain=[('model', '=', 'slide.channel.partner')])
     enroll = fields.Selection([
         ('public', 'Public'), ('invite', 'On Invitation')],
+        compute='_compute_enroll', store=True, readonly=False,
         default='public', string='Enroll Policy', required=True,
         help='Defines how people can enroll to your Course.', copy=False)
     enroll_msg = fields.Html(
         'Enroll Message', help="Message explaining the enroll process",
         default=_get_default_enroll_msg, translate=tools.html_translate, sanitize_attributes=False)
     enroll_group_ids = fields.Many2many('res.groups', string='Auto Enroll Groups', help="Members of those groups are automatically added as members of the channel.")
     visibility = fields.Selection([
-        ('public', 'Open To All'), ('members', 'Members Only')],
-        default='public', string='Visibility', required=True,
+        ('everyone', 'Everyone'),
+        ('logged_user', 'Logged Users'),
+        ('course_attendees', 'Course Attendees')
```

Comment:
The logic behind members (students) hasn't changed either. `members` in eLearning refers to course attendees, and in the current upgrade PR, 'members' are switched to 'course_attendees'. I'm not sure it is necessary.

So, in the same way as above for "members", I would keep the `Selection` key to `"members"` (which can still valuably be **relabeled** as `"Course Attendees"`).